### DISCUSSION TITLE
不用買 Premium！YouTube 不會有廣告的瀏覽器 -- Brave

### DISCUSSION THREAD

**Discussion by 1chooo** [1101-GS4719-C/Python-Community-GS4719-C#1796] [優秀專案]
推薦大家使用 Brave 瀏覽器，我自己用下來是直接擋廣告的（無須安裝 AD Block），是個開源的瀏覽器，核心也跟 Chrome 一樣，所以用習慣 Chrome 的人來說可以無痛切換。

當談到 Brave 瀏覽器時，有幾個關鍵點可以補充說明：（透過 ChatGPT）

1. **隱私保護與安全性：** Brave 瀏覽器以強調使用者隱私保護而聞名。它內建了廣告和跟蹤器的阻擋功能，減少了在線廣告商追蹤用戶的能力。此外，它也提供了HTTPS Everywhere，以確保使用者的連線是加密的，更有助於保護隱私。
2. **基於 Chromium：** Brave 瀏覽器的核心是建立在 Chromium 開源專案之上，這使得它在穩定性和性能方面與 Chrome 有相似的表現。同時，它也允許使用者應用 Chrome 的擴展程式和工具，讓習慣使用 Chrome 的人可以輕鬆切換到 Brave 而無需擔心功能缺失。
3. **獎勵機制：** Brave 瀏覽器提供了一個獎勵機制，讓用戶可以選擇接收基於 Basic Attention Token (BAT) 的獎勵，作為觀看廣告或支持喜歡的內容創作者的方式。這樣的機制讓用戶可以掌握更多的控制權和回饋。
4. **快速且省資源：** 由於去除了大部分廣告和跟蹤器，Brave 瀏覽器可能比其他瀏覽器更快速，同時也可以幫助用戶節省頻寬和裝置資源。
5. **跨平台使用：** Brave 瀏覽器支援多種作業系統，包括 Windows、Mac、Linux、Android 和 iOS，這意味著使用者可以在各種設備上保持一致的瀏覽體驗。

總括來說，Brave 瀏覽器提供了一個強調隱私保護的瀏覽解決方案，同時保持了快速、安全和基於 Chromium 的特點，讓用戶可以享受更流暢的網頁瀏覽體驗。

附上 GitHub Repo 和官網給大家

- GitHub Repo: https://github.com/brave/brave-browser
- Brave Official Site: https://brave.com/

**Comment by uwje** [comment]
這個是個不錯的瀏覽器，但是我認為如果每個人都使用這樣的方式規避廣告，那這樣創作者變無法獲得的預期的收益，使得創作質量降低。

**Comment by chuanmin0317** [comment]
我目前還是選擇訂閱Premium，畢竟如果是六個人一起訂閱的話一個月的價錢也不算貴，而且還有附贈youtube music，因為我也沒有訂閱其他音樂的串流服務，所以對我來說算是付一份錢有兩個功能，還蠻值得的

**Comment by FakeWood** [comment]
對於不想換其他瀏覽器又不想花錢的人，這邊提供一個小技巧，
那就是遇到廣告的時候按上一頁再按下一頁回到影片就不會有廣告了，
搭配快捷鍵就只要按住ALT再快速按一下方向鍵的左右，
這樣不會有廣告了，我自己是覺得蠻方便的，不花時間

**Comment by YANRONG0123** [comment]
我目前是採用網路上教學的uBlock去規避廣告，雖然知道廣告收益也是創作者重要的收入來源，但是不斷看重複的廣告真的令人感到相當頭痛，也不想把錢定期的花在YT上，畢竟只是為了某些需求而使用。

**Comment by YifenLin** [comment]
我自己是安裝AdBlock，因為我自己使用YT的習慣比較像是一陣一陣的，寒暑假使用率高，學期間使用率低，這樣的話，Premium的訂閱制度相對來說是比較不划算的，雖然能夠感受到安裝AdBlock如果遇到廣告還是要等一下下讓他擋掉的時間，不過以我的使用習慣來說已經堪用。

**Comment by poiu6fi** [comment]
我也是用brave，操作起來跟chrome差不多又沒有廣告，還會記錄之前瀏覽的頁面，使用起來很舒服

**Comment by CtyceT** [comment]
雖然手機我也有裝Brave瀏覽器，但我用自己的電腦和手機時還是會使用Chrome，但純粹只是因為習慣。仔細想來Chrome能做的事其實Brave也能做到甚至更好，或許我該改改自己的使用習慣了。
